### PR TITLE
Suppressed / Fixed excessive comments made by javadoc

### PR DESCRIPTION
**Pull request by aashir21**
Signed-off-by: Aashir Siddiqui <aashir_sidiki@hotmail.com>

# Why ?
See issue https://github.com/galasa-dev/projectmanagement/issues/1249

### CODE REVIEW

File: galasa-managers-parent/galasa-managers-cicsts-parent/dev.galasa.cicsts.manager/src/main/java/dev/galasa/cicsts/ICemt.java [hunk 1]
```diff
@@ -16,7 +16,7 @@ public interface ICemt {
     /** 
      * Inquire a CEMT resource using the resource type and name.
      * This does not support inquiries of multiple resources at once. 
-     * Will return {@link null} if the resource is not found.
+     * Will return null if the resource is not found.
```
Comment (techcobweb) [inline galasa-managers-parent/galasa-managers-cicsts-parent/dev.galasa.cicsts.manager/src/main/java/dev/galasa/cicsts/ICemt.java:19]: This comment belongs in the @return statement.

File: galasa-managers-parent/galasa-managers-cicsts-parent/dev.galasa.cicsts.manager/src/main/java/dev/galasa/cicsts/IExecInterfaceBlock.java [hunk 2]
```diff
@@ -51,7 +51,6 @@ public interface IExecInterfaceBlock {
 
     /**
      * Returns the value of the EIBCPOSN field in the EIB.
```
Comment (techcobweb) [inline galasa-managers-parent/galasa-managers-cicsts-parent/dev.galasa.cicsts.manager/src/main/java/dev/galasa/cicsts/IExecInterfaceBlock.java:53]: Move this into the @return area ?

File: galasa-managers-parent/galasa-managers-comms-parent/dev.galasa.http.manager/src/main/java/dev/galasa/http/HttpClientResponse.java [hunk 3]
```diff
@@ -240,11 +240,11 @@ public static HttpClientResponse<String> textResponse(CloseableHttpResponse http
     }
 
     /**
-     * Create an {@link HttpClientResponse} with a {@link JSONObject} content type
+     * Create an {@link HttpClientResponse} with a &lt;com.google.gson.JsonObject&gt; content type
```
Comment (techcobweb) [inline galasa-managers-parent/galasa-managers-comms-parent/dev.galasa.http.manager/src/main/java/dev/galasa/http/HttpClientResponse.java:243]: Don't think you need either &lt; or &gt;

File: galasa-managers-parent/galasa-managers-comms-parent/dev.galasa.http.manager/src/main/java/dev/galasa/http/HttpClientResponse.java [hunk 4]
```diff
@@ -240,11 +240,11 @@ public static HttpClientResponse<String> textResponse(CloseableHttpResponse http
...
      * from an {@link HttpResponse}.
      * 
      * @param httpResponse
-     * @return - {@link HttpClientResponse} with a {@link JSONObject} content type
+     * @return - {@link HttpClientResponse} with a &lt;com.google.gson.JsonObject&gt; content type
```
Comment (techcobweb) [inline galasa-managers-parent/galasa-managers-comms-parent/dev.galasa.http.manager/src/main/java/dev/galasa/http/HttpClientResponse.java:247]: same as above.

File: galasa-managers-parent/galasa-managers-comms-parent/dev.galasa.http.manager/src/main/java/dev/galasa/http/HttpClientResponse.java [hunk 5]
```diff
@@ -253,14 +253,14 @@ public static HttpClientResponse<JsonObject> jsonResponse(CloseableHttpResponse
...
-     * Create an {@link HttpClientResponse} with a {@link JSONObject} content type
+     * Create an {@link HttpClientResponse} with a &lt;com.google.gson.JsonObject&gt; content type
      * from an {@link HttpResponse}. If contentOnBadResponse is true, an attempt
      * will be made to retrieve the content even on a non 200 status code, otherwise
      * the content will be null in such an instance.
      * 
      * @param httpResponse
      * @param contentOnBadResponse
-     * @return - {@link HttpClientResponse} with a {@link JSONObject} content type
+     * @return - {@link HttpClientResponse} with a &lt;com.google.gson.JsonObject&gt; content type
```
Comment (techcobweb) [inline galasa-managers-parent/galasa-managers-comms-parent/dev.galasa.http.manager/src/main/java/dev/galasa/http/HttpClientResponse.java:263]: same as above. Use <code>...</code> if you want to make it look different.

File: galasa-managers-parent/galasa-managers-testingtools-parent/dev.galasa.jmeter.manager/src/main/java/dev/galasa/jmeter/IJMeterSession.java [hunk 6]
```diff
@@ -60,7 +60,7 @@ public interface IJMeterSession {
      * JMX-files have to be prepared for this by replacing ALL the $P__VARIABLE notations with $VARIABLE notations
      * Then by adding neccesary parameters inside a HashMap with it, it is possible to change the target of a test dynamically inside the test
      * Example:
```
Comment (techcobweb) [inline galasa-managers-parent/galasa-managers-testingtools-parent/dev.galasa.jmeter.manager/src/main/java/dev/galasa/jmeter/IJMeterSession.java:62]: Nope. Use this:
/**
* <PRE>
HashMap<String,Object> map = new HashMap<String,Object>();
map.put("HOST", "galasa.dev");
session.setChangedParametersJmxFile(jmxStream, map);
</PRE>
* @param jmxStream
*/

File: galasa-managers-parent/galasa-managers-zos-parent/dev.galasa.zos.manager/src/main/java/dev/galasa/zosbatch/IZosBatchJob.java [hunk 7]
```diff
@@ -92,7 +92,7 @@ public static JobStatus valueOfLabel(String jobStatus) {
     
     /**
      * The batch job completion return code, e.g.<br>
-     * <code>CC 0000</code>, <code>CC 0020</code>, <code>JCL ERROR</code>, <code>ABEND S0C4/code> etc.<br>
+     * &lt;code&gt;CC 0000&lt;/code&gt;, &lt;code&gt;CC 0020&lt;/code&gt;, &lt;code&gt;JCL ERROR&lt;/code&gt;, &lt;code&gt;ABEND S0C4&lt;/code&gt; etc.&lt;br&gt;
```
Comment (techcobweb) [inline galasa-managers-parent/galasa-managers-zos-parent/dev.galasa.zos.manager/src/main/java/dev/galasa/zosbatch/IZosBatchJob.java:95]: Nope. `<code>123</code>` renders the numbers 123 in the monospace font.
You are rendering them in the normal font with `<` and `>` characters.
Needs reverting I think.

File: galasa-managers-parent/galasa-managers-zos-parent/dev.galasa.zosprogram.manager/src/main/java/dev/galasa/zosprogram/ZosProgram.java [hunk 8]
```diff
@@ -96,7 +96,7 @@ public enum Language {
          */
         PL1(".pl1"),
         /**
-         * Assembler program with <code>.asm<\code> file extension 
+         * Assembler program with &lt;code&gt;.asm&lt;\code&gt; file extension
```
Comment (techcobweb) [inline galasa-managers-parent/galasa-managers-zos-parent/dev.galasa.zosprogram.manager/src/main/java/dev/galasa/zosprogram/ZosProgram.java:99]: `<code>xxx</code>` is better

File: galasa-managers-parent/galasa-managers-zos-parent/dev.galasa.zos.manager/src/main/java/dev/galasa/zosbatch/IZosBatchJob.java (hunk 7 above)
Comment (aashir21) [reply galasa-managers-parent/galasa-managers-zos-parent/dev.galasa.zos.manager/src/main/java/dev/galasa/zosbatch/IZosBatchJob.java:95]: or use quotes.

**Comment by galasa-team** [comment]
Build successful

**Review by techcobweb — APPROVED** [review]
Looks good.

**Pull request merged**